packages/react-sdk/src/App.tsx

```diff
@@ -54,5 +54,9 @@ export const App = ({ layoutProps }: AppProps) => {
     );
   }
 
-  return <Layout {...layoutProps} />;
+  return (
+    <div id="widget-root">
```

Comment:
Can we stick the ID to some element that already exist?

Reply:
will need to check that. In theory yes

Reply:
Yes this worked